packages/lodestar/src/chain/attestation.ts

```diff
@@ -101,7 +101,9 @@ export class AttestationProcessor implements IAttestationProcessor {
     if(justifiedCheckpoint.epoch > GENESIS_EPOCH) {
       const justifiedBlock =
         this.forkChoice.getBlockSummaryByBlockRoot(justifiedCheckpoint.root.valueOf() as Uint8Array);
-      checkpointState = await this.db.stateCache.get(justifiedBlock.stateRoot);
+      if (justifiedBlock) {
+        checkpointState = await this.db.stateCache.get(justifiedBlock.stateRoot);
```

Comment:
I think we should throw error here or state will be null later on

Reply:
actually I think this will never happen, we should always get a not-null justified checkpoint from forkchoice